packages/visx-delaunay/yarn.lock

```diff
@@ -0,0 +1,92 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
since this is a monorepo, we can delete this file and use only the `yarn.lock` in the monorepo root directory. if you delete this and run `yarn` in the `root` it should install deps as expected.